src/components/OptionRow.tsx

```diff
@@ -16,169 +16,164 @@ import Icon from './Icon';
 
     /** Whether we should show the selected state */
-    showSelectedState: PropTypes.bool,
+    showSelectedState?: boolean;
 
     /** Whether to show a button pill instead of a tickbox */
-    shouldShowSelectedStateAsButton: PropTypes.bool,
+    shouldShowSelectedStateAsButton?: boolean;
 
     /** Text for button pill */
-    selectedStateButtonText: PropTypes.string,
+    selectedStateButtonText?: string;
 
     /** Callback to fire when the multiple selector (tickbox or button) is clicked */
-    onSelectedStatePressed: PropTypes.func,
+    onSelectedStatePressed?: (option: OptionData) => void;
 
     /** Whether we highlight selected option */
-    highlightSelected: PropTypes.bool,
+    highlightSelected?: boolean;
 
     /** Whether this item is selected */
-    isSelected: PropTypes.bool,
+    isSelected?: boolean;
 
     /** Display the text of the option in bold font style */
-    boldStyle: PropTypes.bool,
+    boldStyle?: boolean;
 
     /** Whether to show the title tooltip */
-    showTitleTooltip: PropTypes.bool,
+    showTitleTooltip?: boolean;
 
     /** Whether this option should be disabled */
-    isDisabled: PropTypes.bool,
+    isDisabled?: boolean;
 
     /** Whether to show a line separating options in list */
-    shouldHaveOptionSeparator: PropTypes.bool,
+    shouldHaveOptionSeparator?: boolean;
 
     /** Whether to remove the lateral padding and align the content with the margins */
-    shouldDisableRowInnerPadding: PropTypes.bool,
+    shouldDisableRowInnerPadding?: boolean;
 
     /** Whether to prevent default focusing on select */
-    shouldPreventDefaultFocusOnSelectRow: PropTypes.bool,
+    shouldPreventDefaultFocusOnSelectRow?: boolean;
 
     /** Whether to wrap large text up to 2 lines */
-    isMultilineSupported: PropTypes.bool,
+    isMultilineSupported?: boolean;
 
-    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+    style?: StyleProp<TextStyle>;
 
-    ...withLocalizePropTypes,
+    backgroundColor?: string;
 };
 
-const defaultProps = {
-    hoverStyle: undefined,
-    showSelectedState: false,
-    shouldShowSelectedStateAsButton: false,
-    selectedStateButtonText: 'Select',
-    onSelectedStatePressed: () => {},
-    highlightSelected: false,
-    isSelected: false,
-    boldStyle: false,
-    showTitleTooltip: false,
-    onSelectRow: undefined,
-    isDisabled: false,
-    optionIsFocused: false,
-    isMultilineSupported: false,
-    style: null,
-    shouldHaveOptionSeparator: false,
-    shouldDisableRowInnerPadding: false,
-    shouldPreventDefaultFocusOnSelectRow: false,
-};
-
-function OptionRow(props) {
+function OptionRow({
+    option,
+    onSelectRow,
+    style,
+    hoverStyle,
+    isDisabled: isOptionDisabled = false,
+    isMultilineSupported = false,
+    shouldShowSelectedStateAsButton = false,
+    highlightSelected = false,
+    selectedStateButtonText = 'Select',
```

Comment:
Shouldn't this be in the translation file? 🤔

Reply:
@fabioh8010 I've not found it there... Since `selectedStateButtonText` is used only when `shouldShowSelectedStateAsButton` is `true`, and in the default values it's `false` maybe we can turn the default value into an empty string.


Another suggestion is to remove the default value at all, add _Select_ into translation vocabularies and update the code to be
```
text={selectedStateButtonText ?? translate('common.select')}
```

Reply:
I think is best to add this string to the translations!